### PR TITLE
init: readme

### PR DESCRIPTION
**Pull request by MarcelCoding**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    inputs.flake-utils.follows = "flake-utils";
+  };
+};
+```
+
+### NixOS
+
+```nix
+# home.nix
+mports = [ inputs.nvim.nixosModules.nvim ];
```
Comment (tanneberger) [inline README.md:32]: *imports

File: README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+```nix
+# flake.nix
+inputs = {
+  nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.05";
+  flake-utils.url = "github:numtide/flake-utils";
+  home-manager = {
+    url = "github:nix-community/home-manager/release-24.05";
+    inputs.nixpkgs.follows = "nixpkgs";
+  };
+
```
Comment (tanneberger) [inline README.md:14]: missing nixvim input

File: README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  };
+};
+```
+
+### NixOS
+
+```nix
+# home.nix
+imports = [ inputs.nvim.nixosModules.nvim ];
+programs.nixvim.enable = true;
```
Comment (SuperSandro2000) [inline README.md:38]:
```suggestion
{
  imports = [ inputs.nvim.nixosModules.nvim ];

  programs.nixvim.enable = true;
}
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    inputs.nixvim-stable.follows = "nixvim";
+    inputs.flake-utils.follows = "flake-utils";
+  };
+};
+```
+
+### NixOS
+
+```nix
+# home.nix
```
Comment (SuperSandro2000) [inline README.md:36]:
```suggestion
# configuration.nix
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+imports = [ inputs.nvim.nixosModules.nvim ];
+programs.nixvim.enable = true;
+```
+
+### HomeManager
+
+```nix
+# home.nix
+imports = [ inputs.nvim.homeManagerModules.nvim ];
+programs.nixvim.enable = true;
```
Comment (SuperSandro2000) [inline README.md:46]:
```suggestion
# home.nix
{
  imports = [ inputs.nvim.homeManagerModules.nvim ];

  programs.nixvim.enable = true;
}
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+### Package
+
+```nix
+environment.systemPackages = [ inputs.nvim.packages.x86_64-linux.nixvim ];
```
Comment (SuperSandro2000) [inline README.md:52]:
```suggestion
{
  environment.systemPackages = [ inputs.nvim.packages.x86_64-linux.nixvim ];
}
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    url = "github:NuschtOS/nvim.nix";
+    inputs.nixpkgs.follows = "nixpkgs";
+    inputs.nixpkgs-stable.follows = "nixpkgs";
+    inputs.home-manager.follows = "home-manager";
+    inputs.home-manager-stable.follows = "home-manager";
+    inputs.nixvim.follows = "nixvim";
+    inputs.nixvim-stable.follows = "nixvim";
+    inputs.flake-utils.follows = "flake-utils";
+  };
+};
```
Comment (SuperSandro2000) [inline README.md:30]:
```suggestion
inputs = {
  nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.05";
  nixvim = {
    url = "github:nix-community/nixvim/nixos-24.05";
    inputs.nixpkgs.follows = "nixpkgs";
  };
 
  nvim = {
    url = "github:NuschtOS/nvim.nix";
    inputs.nixpkgs.follows = "nixpkgs";
  };
};
```

**Pull request merged**